apps/taxes/.env.example

```diff
@@ -8,12 +8,15 @@ PORT=
 VERCEL_URL=
 REST_APL_ENDPOINT=
 REST_APL_TOKEN=
-ALLOWED_DOMAIN
+ALLOWED_DOMAIN=
+GITHUB_SHA=
+VERCEL=
+VERCEL_GIT_COMMIT_SHA=
```

Comment:
Do we need this? These variables should be auto populated in CI

Reply:
removed